src/tests/test_utils/common_test_utils/src/node_builders/convolution.cpp

```diff
@@ -32,7 +32,7 @@ std::shared_ptr<ov::Node> make_convolution(const ov::Output<Node>& in,
     if (!filter_weights.empty()) {
         filter_weights_node = std::make_shared<ov::op::v0::Constant>(type, filter_weights_shape, filter_weights);
     } else {
-        auto tensor = create_and_fill_tensor(type, filter_weights_shape);
+        auto tensor = create_and_fill_tensor(type, filter_weights_shape, 9, 1);
```

Comment:
Why the change is required?

Reply:
to change default ranks in the case?

Reply:
@iefode Sorry, I didn't get what you mean

Reply:
I totally agree with @vurusovs  and do not understand why you have changed the ranges.
In case we really need to update tensor filling, please update `generate_inputs` for Convolution

Reply:
@iefode this is not about model parameters and inputs. This is about constants creation. We cannot operate with constants in generate_inputs. 
Indeed I made a mistake when I refactored these builders. And This change just fixes them.

If you want, we can have a quick call to clarify it.